packages/dashboard-e2e/tests/ui-interactions/submit-task.test.ts

```diff
@@ -4,7 +4,7 @@ describe('submit task', () => {
   it('can submit loop task', async () => {
     const appBar = await getAppBar();
     await (await appBar.$('button[aria-label="Tasks"]')).click();
-    await (await $('button[aria-label="Create Task"]')).click();
+    await (await appBar.$('button[aria-label="new task"]')).click();
```

Comment:
thanks! could you add an additional e2e test that requests a new task from the `Robot` tab, to verify that we can make new requests from anywhere in the dashboard

Reply:
Oh, man. I'm sorry, I forgot that we mentioned it per VC. 
I've added another file to test it to make it more organized.